### PR TITLE
Tool to generate sample spreadsheet data

### PR DESCRIPTION
**Pull request by GeordieEK**
Data generation tool

### CODE REVIEW

**Review by GeordieEK — COMMENTED** [review]
Data generator

**Review by marmaladian — CHANGES_REQUESTED** [review]
One enhancement before we merge...

File: dataGenerator.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import random
+
+if __name__ == '__main__':
+    '''
+    dataGen generates a text file of spreadsheet values
+
+    @param size: the size of the spreadsheet
```
[the remark this answers is not in the document]
Comment (GeordieEK) [reply dataGenerator.py:7]: yes :()

[the remark this answers is not in the document]
Comment (GeordieEK) [reply dataGenerator.py]: Done! Can also now be run from CLI with new and improved arguments

**Review by GeordieEK — COMMENTED** [review]
Requested features added

**Review by GeordieEK — COMMENTED** [review]
Requested changes added

**Review by marmaladian — APPROVED** [review]
Nice work

**Pull request merged**